src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -458,7 +458,7 @@ public bool Add(T item)
         /// Searches the set for a given value and returns the equal value it finds, if any.
         /// </summary>
         /// <param name="equalValue">The value to search for.</param>
-        /// <param name="actualValue">The value from the set that the search found, or the original value if the search yielded no match.</param>
+        /// <param name="actualValue">The value from the set that the search found, or the default value for the type of the <paramref name="actualValue"/> parameter when the search yielded no match.</param>
```

Comment:
Can we just say the default value of T rather than the default value for the type of the actualValue parameter?

Reply:
Sure, I like that more personally. I made it like this to match the [Dictionary documentation](https://msdn.microsoft.com/en-us/library/bb347013(v=vs.110).aspx) that uses this verbage.